app/assets/stylesheets/application.scss

```diff
@@ -68,7 +74,7 @@ img {
     font-size: 3em;
     padding-bottom: 20px;
     a {
-      color: white;
+      color: #1b440c;
```

Comment:
Color literals like `#1b440c` should only be used in variable declarations; they should be referred to via variable everywhere else.